### PR TITLE
Pt 164259596 generate format op

### PR DESCRIPTION
**Pull request by happi**
Generate the pretty printer for fate ops form spec.

### CODE REVIEW

File: src/aeb_fate_asm.erl [hunk 1]
```diff
@@ -33,14 +48,15 @@
...
+%%%       6. Map
 %%%          {}
+%%%          { 1 => { "foo" => true, "bar" => false}
 %%%       7. Lists
 %%%          []
 %%%          [1, 2]
 %%%       8. Bit field
 %%%          <000>
-%%%          <1010>
+%%%          <1010 1010>
```
Comment (ThomasArts) [inline src/aeb_fate_asm.erl]: is this space allowed?

File: src/aeb_fate_data.erl [hunk 2]
```diff
@@ -133,30 +133,25 @@ format(?FATE_TRUE) -> "true";
...
+      lists:join("| ", [integer_to_list(Size), integer_to_list(Tag) |
+                        [format(E) || E <- erlang:tuple_to_list(T)]]),
+     " |)"];
 format(M) when ?IS_FATE_MAP(M) ->
-    "#{ "
-        ++ format_kvs(maps:to_list(?FATE_MAP_VALUE(M)))
-        ++" }";
-format(?FATE_ADDRESS(Address)) -> address_to_base58(Address);
+    ["{ ", format_kvs(maps:to_list(?FATE_MAP_VALUE(M))), " }"];
+format(?FATE_ADDRESS(Address)) -> ["#", address_to_base58(Address)];
```
Comment (ThomasArts) [inline src/aeb_fate_data.erl]: seems not yet to handle "bits"

File: src/aeb_fate_asm.erl (hunk 1 above)
Comment (happi) [reply src/aeb_fate_asm.erl]: Yes, though it might be nice to be able to group bit fields.

File: src/aeb_fate_data.erl (hunk 2 above)
Comment (happi) [reply src/aeb_fate_data.erl]: They will turn into an int for now... Ill fix.

**Pull request merged**